CGPs/cgp-0071.md

```diff
@@ -0,0 +1,132 @@
+---
+cgp: 71
+title: Addings USDC oracles
+date-created: 2023-02-16
+author: Roman Croessmann(@rcroessmann)
```

Comment:
This needs to be wrapped in single quotes to avoid a yaml error.

Reply:
Thanks, done!

Reply:
thanks